### PR TITLE
minor log tweaks

### PR DESCRIPTION
**Pull request by igorbernstein2**

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 1]
```diff
@@ -546,6 +547,9 @@ public ManagedChannel create() throws IOException {
   public static ManagedChannel createNettyChannel(String host,
       BigtableOptions options, ClientInterceptor ... interceptors) throws SSLException {
 
+    LOG.info("Creating new channel for %s", host);
+    LOG.debug(Throwables.getStackTraceAsString(new Throwable()));
```
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: wait... this is expensive.  Please add an if statement around this.

Comment (igorbernstein2) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: Updated, PTAL

Comment (sduskis) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: we can also add a new method in the `Logger` to do this.

Comment (igorbernstein2) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: I really don't have strong feelings on this. What did you have in mind?

Comment (sduskis) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: First off, why do you want this?

Comment (sduskis) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: Can you please remove this line, and we'll revisit this in a separate PR?

**Pull request merged**